form/root_storage/root_tbranch_write_container.cpp

```
@@ -130,3 +130,11 @@ void ROOT_TBranch_Write_ContainerImp::commit()
   m_tree->SetEntries(m_branch->GetEntries());
   return;
 }
+
+std::uint64_t ROOT_TBranch_Write_ContainerImp::getEntryCount()
```

Cppcheck (2.21.0)

[style] The function 'getEntryCount' is never used.

(unusedFunction)